twake/frontend/src/app/scenes/Client/Popup/WorkspaceParameter/WorkspaceParameter.js

```diff
@@ -2,8 +2,8 @@ import React, { Component } from 'react';
 
 import Languages from 'services/languages/languages';
 import Collections from 'app/services/Depreciated/Collections/Collections.js';
-import LoginService from 'services/login/login.js';
-import loginService from 'services/login/login.js';
+import LoginService from 'app/services/login/LoginService';
+import loginService from 'app/services/login/LoginService';
```

Comment:
New duplication

Reply:
Looks like my search and replace worked well :)